README.md

```diff
@@ -32,7 +32,23 @@ To install linters and other dependencies run:
 npm install
 ```
 
-To go live on local server run:
+To setup Firebase Authentication
+
+- [Firebase Authentication Tutoria](https://www.youtube.com/watch?v=cZAnibwI9u8)
+
+To create Firestore Database
+
+- [Firestore Database Tutoria](https://www.youtube.com/watch?v=TW02hwhBvo4)
+- Create each product on the firestore database following the schema below
+  ![Alt text](public/schema.png)
+
+To Store your firebase credentials
+
+- Create a file named .env in the root folder
+- Paste your Firebase credential following the pattern below
+- ![.env file content](public/env.png)
```

Comment:
[Suggestiom]
The standard to providing `.env` sample is 

1. create a file named `.env.sample`
2. paste the requried env keys e.g.

```
NEXT_PUBLIC_API_KEY=<REPLACE_ME>
NEXT_PUBLIC_AUTH_DOMAIN=<REPLACE_ME>
NEXT_PUBLIC_PROJECT_ID=<REPLACE_ME>
NEXT_PUBLIC_STORAGE_BUCKET=<REPLACE_ME>
NEXT_PUBLIC_MESSAGING_SENDER_ID=<REPLACE_ME>
NEXT_PUBLIC_APP_ID=<REPLACE_ME>
```

This way, in your readme, inform devs to 
1. Create a file named `.env`
2. Copy content from `.env.sample`
3. assign appropriate values to the keys in `.env`

P.S.  When you adopt this feedback, please remove `env.png`